test/serializer/additional-functions/ResidualFunctionBindingsMutatedByOptimizedFunctionRegressionTest.js

```diff
@@ -1,3 +1,5 @@
+// expected Warning: PP1007
+// expected FatalError: PP0013
```

Comment:
PP0013 is about `for` loops. There is no `for` loop in this test. What is going on?